tests/dtm/test_rgw_s3_process_restart.py

```diff
@@ -426,8 +431,9 @@ def test_continuous_read_during_rgw_s3_restart(self):
         self.dtm_obj.perform_write_op(bucket_prefix=self.bucket_name,
                                       object_prefix=self.object_name,
                                       no_of_clients=self.test_cfg['clients'],
-                                      no_of_samples=self.test_cfg['test_42247']['nsamples'],
-                                      log_file_prefix=log_file_prefix, queue=que)
+                                      no_of_samples=self.test_cfg['test_42247']['samples'],
+                                      log_file_prefix=log_file_prefix, queue=que,
+                                      loop=self.test_cfg['loop_count'])
```

Comment:
@ApurwaMohite We were reading the same data in multiple loops. hence had added loop count in reads.